.githooks/pre-push.bash

```diff
@@ -2,10 +2,14 @@
 
 set -euo pipefail
 
-# Check if we actually have commits to push
-commits=$(git log @{u}..)
-if [ -z "$commits" ]; then
-    exit 0
+# Get current branch name
+current=$(git rev-parse --abbrev-ref HEAD)
+if git branch -r | grep "^  origin/${current}$" &> /dev/null; then
```

Comment:
If someone renames the remote differently than `origin` it probably would not work. Maybe to replace `origin` with `$1` which is a remote?